### PR TITLE
return logits

### PR DESCRIPTION
**Pull request by mwitiderrick**

### CODE REVIEW

**Comment by mwitiderrick** [comment]
```
from deepsparse import Pipeline
cv_pipeline = Pipeline.create(
  task='image_classification',
  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
  top_k=3,
  return_logits = True,
)
input_image = "cheetah.jpeg" # path to input image
inference = cv_pipeline(images=input_image)
inference

# ImageClassificationOutput(labels=None, scores=None, logits=array([[ 1.8277559 ,  4.486894  ,  2.4551017 ,  4.777242  ,  # 2.744733  ,.... 8.45131   ]],
 #     dtype=float32))
```
